### PR TITLE
Added an autogeneration using gir.

### PR DESCRIPTION
**Pull request by RazrFalcon**

### CODE REVIEW

File: src/widgets/font_description.rs [hunk 1]
```diff
@@ -30,6 +30,20 @@ impl FromGlibPtr<*mut ffi::PangoFontDescription> for FontDescription {
...
+        assert!(!tmp.is_null());
+        FontDescription(tmp as *mut _)
+    }
+
+    unsafe fn from_glib_full(ptr: *const ffi::PangoFontDescription) -> Self {
+        assert!(!ptr.is_null());
+        FontDescription(ptr as *mut _)
+    }
+}
+
```
Comment (EPashkin) [inline src/widgets/font_description.rs]: Too many empty lines

Comment (RazrFalcon) [inline src/widgets/font_description.rs]: You want me to remove [this line](https://github.com/RazrFalcon/pango/blob/autogen_support/src/widgets/font_description.rs#L46)?

Comment (EPashkin) [inline src/widgets/font_description.rs]: Yes

Comment (RazrFalcon) [inline src/widgets/font_description.rs]: Done.

Comment (EPashkin) [inline src/widgets/font_description.rs]: No needed extra commit for this. Just squash.

Comment (EPashkin) [inline src/widgets/font_description.rs]: Or ammend

Comment (RazrFalcon) [inline src/widgets/font_description.rs]: But I've already pushed it. I can't squash without `--force` now. Or can?

Comment (EPashkin) [inline src/widgets/font_description.rs]: You can push --force your branch without problem. Github handle this and update PR.

Comment (RazrFalcon) [inline src/widgets/font_description.rs]: Done.

**Comment by EPashkin** [comment]
@RazrFalcon, All good now for me, thanks.
Now waiting for CI finished and @GuillaumeGomez take look at this.

**Comment by GuillaumeGomez** [comment]
Well, I don't have much to say, you already did all the work @EPashkin! ;)

Thanks @RazrFalcon!

**Pull request merged**